Dockerfile.preflight

```diff
@@ -0,0 +1,13 @@
+# This builds a preflight playbook image for use with Docker.
+# Look at playbooks/byo/preflight/README.md for more information.
+# docker build -t docker.io/sosiouxme/ocp_preflight_playbook -f Dockerfile.preflight .
+
+FROM docker.io/aweiteka/playbook2image
```

Comment:
What's the advantage of building on top of that image? I think we don't need to have anything to do with S2I.

Maybe it's simpler to base on `openshift/base-centos7` (parent of  `aweiteka/playbook2image`), then install ansible and dependencies, and set `ansible-playbook ...` as the ENTRYPOINT/CMD?